README.md

```diff
@@ -1,6 +1,6 @@
 # React_ToDoList
 Creating a To-DoList in react using express, postgresql
-@starrpatty28 @brianb1 @sashajustice
+ colaborators @starrpatty28 @brianb1 @sashajustice
 # To Do List
 ## Description
 
```

Comment:
Maybe do a bit more on your description before submitting a pr so there's more to review...